etc/profile-a-l/electron-mail.profile

```diff
@@ -1,55 +1,43 @@
 # Firejail profile for electron-mail
```

Comment:
```suggestion
# Firejail profile for ElectronMail
```

Clarify that there is an actual program called "ElectronMail".

Reply:
It is kind of confusing. I think tutanota and proton started with the same code and are running now slightly different flavors of it. I would say let's make a new ElectronMail.profile and redirect it to electron-mail.profile, the same way we do it with .gimp-2.10.profile and gimp.profile. We'll fix it when people start complaining.

Reply:
> electron-mail is the actual command for the app. Not ElectronMail

Yes, hence the filename is `electron-mail.profile`. The name used at user-facing places (e.g. .desktop files) is ElectronMail AFAIK. Which is IMHO the better name for this place but rarely used.